### PR TITLE
Adding manifest deprication warning to execution environments doc

### PR DESCRIPTION
**Pull request by annibond**
Added warnings about the depreciation of RHSM manifests & manifest lists, interchangeably called subscription allocations, to:

- Assembly chapters 5. Populating your private automation hub container registry
- Procedure chapter 5.1 Pulling images for use in automation hub

In 5.1, I also added reference links for
- Added links for Satellite Release Dates, as it relates to the one exception to the currently existing depreciation
- Transition of Red Hat's subscription services to console.redhat.com from the Customer Portal

Because the use of consoleDOT is new to the user potentially, I have also added two steps to the procedure in 5.1 addressing that.

### CODE REVIEW

**Review by g-murray — COMMENTED** [review]
3 small comments, besides that LGTM, let me know when you're ready for me to squash and merge

File: downstream/modules/hub/proc-obtain-images.adoc [hunk 1]
```diff
@@ -8,11 +8,26 @@
...
+
+* You have permissions to pull images from registry.redhat.io.
+
+* A Red Hat account with Simple Account Access enabled.
 
 .Procedure
 
+. If you need to access your manifest for your container images log in to link:https://console.redhat.com/subscriptions/manifests[Red Hat Console].
+
+. Click the three-dot menu for the manifest you need for your container images, and click *Export manifest*.
```
[the remark this answers is not in the document]
Comment (annibond) [reply downstream/modules/hub/proc-obtain-images.adoc:29]: That's a great bit of information to learn, thank you Greg!

File: downstream/modules/hub/proc-obtain-images.adoc [hunk 2]
```diff
@@ -8,11 +8,26 @@
...
+
+New Red Hat accounts automatically use Simple Content Access for their subscription tooling. New Red Hat accounts and existing Satellite customers who can connect to Red Hat’s servers can find their manifests at `console.redhat.com`.
+====
+
 .Prerequisites
-You have permissions to pull images from registry.redhat.io.
+
+* You have permissions to pull images from registry.redhat.io.
+
+* A Red Hat account with Simple Account Access enabled.
```
Comment (hherbly) [inline downstream/modules/hub/proc-obtain-images.adoc:23]: should be Simple Content Access, not Simple Account Access

File: downstream/modules/hub/proc-obtain-images.adoc [hunk 3]
```diff
@@ -43,4 +58,6 @@ $ podman images
 [role="_additional-resources"]
 .Additional resources
 
-* See link:https://redhat-connect.gitbook.io/catalog-help/[Red Hat Ecosystem Catalog Help] for information on registering and getting images.
+* See link:redhat-connect.gitbook.io/catalog-help/[Red Hat Ecosystem Catalog Help] for information on registering and getting images.
+
+* See link:{BaseURL}subscription_central/1-latest/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected Satellite Server] to learn more about the changes coming to Red Hat subscription tooling
```
Comment (hherbly) [inline downstream/modules/hub/proc-obtain-images.adoc:63]: I believe there needs to be a slash right after the base URL attribute for this to work--so it should be 
{BaseURL}/subscription_central/1-latest/html/creating_and_managing_manifests_for_a_connected_satellite_server/index

**Review by hherbly — APPROVED** [review]
LGTM!

**Pull request merged**